src/Common/test/Common.Expression.Test/Spring/SpelReproTests.cs

```diff
@@ -11,6 +11,11 @@
 using Steeltoe.Common.Util;
 using Xunit;
 
+// ReSharper disable InconsistentNaming
+#pragma warning disable S100 // Methods and properties should be named in PascalCase
+#pragma warning disable SA1307 // Accessible fields should begin with upper-case letter
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
Should these be turned back on later in the file?

Reply:
No, all preprocessor defines automatically go out of scope at the end of the file they were declared in.